### ISSUE TITLE
corrupt config files are silently ignored even with -c

### ISSUE THREAD

**Issue by rbtcollins** [wata727/tflint#202]
I tried to pass -c /path/to/an/explicit/config file but it seemed to have no effect: no error, and the config had no impact.

Perhaps I'm misunderstanding the intent of the option?

**Comment by wata727** [comment]
Hi @rbtcollins, the `-c` option works even if using absolute paths. You can pass the name of the configuration file instead of `.tflint.hcl` for this option.

If it doesn't work as intended, try the `--debug` option and paste the output.

**Comment by wata727** [comment]
Ah, yes. That is a bug... Thanks for your report.
But, maybe this bug is fixed in v0.6.0. I confirmed this behavior with the following corrupt config:

```hcl
config {
```

```
$ tflint -c /tmp/.tflint.hcl
/tmp/.tflint.hcl:2,1-1: Attribute or block definition required; An attribute or block definition is required here.
```

However, this error is still difficult to understand, so there is room for improvement.
What version of TFLint do you use?

**Comment by wata727** [comment]
Sorry, I was misunderstanding... v0.6.0 still has this bug. But, it is fixed in the current master branch. This fix will be included in the next release.

**Comment by rbtcollins** [comment]
Great, thanks. Seems like this can be closed then.

**Comment by rbtcollins** [comment]
(I agree that there is room for improvement, but my issue was the lack of signalling, not the beauty of the message :) )
